mmv1/third_party/terraform/utils/compute_operation.go.erb

```diff
@@ -161,10 +161,20 @@ func computeOrgOperationWaitTimeWithResponse(config *Config, res interface{}, re
 type ComputeOperationError compute.OperationError
 
 func (e ComputeOperationError) Error() string {
-	var buf bytes.Buffer
+	buf := bytes.NewBuffer(nil)
 	for _, err := range e.Errors {
-		buf.WriteString(err.Message + "\n")
+		writeOperationError(buf, err)
 	}
 
 	return buf.String()
 }
+
+func writeOperationError(w io.StringWriter, opError *compute.OperationErrorErrors) {
```

Comment:
I think we need to add a dependency for `io` to make this work. Currently the build fails due to missing packages

Reply:
Ack, will fix.

Reply:
Added `io`.